test/e2e/dai.spec.ts

```diff
@@ -30,11 +31,11 @@ describe('DAI @skip-on-coverage', () => {
     dai = sdk.dai;
 
     daiWhale = await wallet.impersonate(daiWhaleAddress);
-    snapshotId = await evm.snapshot.take();
+    snapshot = await helpers.takeSnapshot();
```

Comment:
for some reason, `helpers` here is coming as `undefined` while on the L:33, we use our tooling that also references `helpers` and its working just fine. 

Any clue what might be going on? @turtlemoji @wei3erHase?